src/classes/LD_LeadConvertOverride_CTRL.cls

```diff
@@ -421,11 +445,29 @@ public with sharing class LD_LeadConvertOverride_CTRL {
             // if we created a new company account, affiliate to it.
             if (strAccountId == 'NEW')
                 strAfflAccountId = lcr.getAccountId();
-                
+            
+            // if we created a new company acct and the Lead had a company address, set the new acct's address
+            if (strAccountId == 'NEW' && !String.isBlank(l.CompanyStreet__c)) {
+            	update new Account(
+                    Id=lcr.getAccountId(),
+                    BillingStreet = l.CompanyStreet__c, 
+                    BillingCity = l.CompanyCity__c,
+                    BillingState = l.CompanyState__c,
+                    BillingPostalCode = l.CompanyPostalCode__c,
+                    BillingCountry = l.CompanyCountry__c
+                );
+            }   
             // handle the optional Affiliation to a new or existing Account.
             ID afflAccId = null;
             if (strAfflAccountId == 'NEW') {
-                Account acc = new Account(name=l.Company);
+                Account acc = new Account(
```

Comment:
unfortunately, my org with these enabled is currently in use by me for all my current work.  it is easy to enable/disable in any de org, though.  just search for state & country under setup.  also, Chris & Jason may have finally set up a test org with them enabled.